app/views/members/_form.html.erb

```diff
@@ -98,11 +98,16 @@
       <dd><%= f.text_field :interests, class: "wide-input" %></dd>
     </dl>
 
+    <dl>
+        <dt><%= f.label :prefers_full_name, "Prefer full name:" %></dt>
+        <dd><%= f.check_box :prefers_full_name %></dd>
```

Comment:
likewise I think it is a little more straightforward to say something like `:show_real_names`, as we aren't really showing "full names" in the traditional sense and "prefers" is more ambiguous as to what the result is